### DISCUSSION TITLE
How to formatted tooltip value?

### DISCUSSION THREAD

**Discussion by BigFaceMaster** [wuba/react-native-echarts#94] [Q&A] [answered]
```tsx
const options = {
   tooltip: {
      formatter: params => {
        custom function body
         return 'result'
      }
   }
}
```

**Comment by RafaelCENG** [comment, marked as answer]
You can use valueFormatter. From the echarts docs
```js
 tooltip: {
        trigger: "axis",
        valueFormatter: function (params) {
          return `${params}W`;
        },
      },
```

Now if you want to show only the value for example you can do something like this.
```js
   formatter: function (params) {
          return `${params[0].data}.W`;
        },
```
Also this is working 100% on single LineChart.

**Comment by BigFaceMaster** [comment]
在Tooltip中使用 自定义的 html ，似乎依旧有问题

**Reply by zhiqingchen** [reply]
`html` not supported
https://github.com/wuba/react-native-echarts/issues/16

**Reply by guozi9999** [reply]
这个有支持的计划吗

**Reply by zhiqingchen** [reply]
目前没有

**Reply by baicuhaha** [reply]
为啥 rich: {
          label: {
            align: 'left',
            color: 'rgba(13, 12, 34, 1)', // 设置左侧标签颜色
            fontSize: 10,
            width: 400,
            backgroundColor: 'red',
          },
          value: {
            align: 'right', // 右对齐
            color: 'red', // 设置左侧标签颜色
            fontSize: 10,
            width: 60,
            backgroundColor: 'red',
          },
        },
 formatter: function (params: any) {
          // 自定义浮窗内容
          const data = params[0].data; // 获取K线图数据

          let listData = data.value;
          let open = listData[1] ? parseFloat(listData[1]).toFixed(numm) : '';
          let Close = listData[2] ? parseFloat(listData[2]).toFixed(numm) : '';
          let Low = listData[3] ? parseFloat(listData[3]).toFixed(numm) : '';
          let High = listData[4] ? parseFloat(listData[4]).toFixed(numm) : '';
          if (!params[0].name) {
            return '';
          }
          return (
            '{label|Time:}' +
            `{value|${params[0].name}}\n` +
            `{label|Open:}{value|${open}}\n` +
            `{label|Close:}{value|${Close}}\n` +
            `{label|Low:}{a|${Low}}\n` +
            `{label|High:}{a|${High}}`
          );
        },
rich 语法也不生效。。
